website/docs/r/workloads_sap_three_tier_virtual_instance.html.markdown

```diff
@@ -0,0 +1,546 @@
+          ssh_public_key  = data.tls_public_key.example.public_key_openssh
+        }
+      }
+    }
+
+    database_server_configuration {
+      instance_count = 1
+      subnet_id      = azurerm_subnet.example.id
+      database_type  = "HANA"
+
+      virtual_machine_configuration {
+        virtual_machine_size = "Standard_E16ds_v4"
+
+        image {
+          offer     = "RHEL-SAP-HA"
+          publisher = "RedHat"
+          sku       = "82sapha-gen2"
+          version   = "latest"
+        }
+
+        os_profile {
+          admin_username  = "testAdmin"
+          ssh_private_key = tls_private_key.example.private_key_pem
+          ssh_public_key  = data.tls_public_key.example.public_key_openssh
+        }
+      }
+
+      disk_volume_configuration {
+        volume_name     = "hana/data"
+        number_of_disks = 3
+        size_in_gb      = 128
+        sku_name        = "Premium_LRS"
+      }
+
+      disk_volume_configuration {
+        volume_name     = "hana/log"
+        number_of_disks = 3
+        size_in_gb      = 128
+        sku_name        = "Premium_LRS"
+      }
+
+      disk_volume_configuration {
+        volume_name     = "hana/shared"
+        number_of_disks = 1
+        size_in_gb      = 256
+        sku_name        = "Premium_LRS"
+      }
+
+      disk_volume_configuration {
+        volume_name     = "usr/sap"
+        number_of_disks = 1
+        size_in_gb      = 128
+        sku_name        = "Premium_LRS"
+      }
+
+      disk_volume_configuration {
+        volume_name     = "backup"
+        number_of_disks = 2
+        size_in_gb      = 256
+        sku_name        = "StandardSSD_LRS"
+      }
+
+      disk_volume_configuration {
+        volume_name     = "os"
+        number_of_disks = 1
+        size_in_gb      = 64
+        sku_name        = "StandardSSD_LRS"
+      }
+    }
+
+    resource_names {
+      application_server {
+        availability_set_name = "appAvSet"
+
+        virtual_machine {
+          host_name               = "apphostName0"
+          os_disk_name            = "app0osdisk"
+          virtual_machine_name    = "appvm0"
+          network_interface_names = ["appnic0"]
+
+          data_disk_names = {
+            default = "app0disk0"
+          }
+        }
+      }
+
+      central_server {
+        availability_set_name = "csAvSet"
+
+        load_balancer {
+          name                            = "ascslb"
+          backend_pool_names              = ["ascsBackendPool"]
+          frontend_ip_configuration_names = ["ascsip0"]
+          health_probe_names              = ["ascsHealthProbe"]
+        }
+
+        virtual_machine {
+          host_name               = "ascshostName"
+          os_disk_name            = "ascsosdisk"
+          virtual_machine_name    = "ascsvm"
+          network_interface_names = ["ascsnic"]
+
+          data_disk_names = {
+            default = "ascsdisk"
+          }
```

Comment:
could we update this to reflect the new name/layout of this property?

Reply:
Updated